docs/Other SDL Features/Remote Control Vehicle Features/index.md

```diff
@@ -281,24 +570,50 @@ sdlManager.sendRPC(setInteriorVehicleData);
 ```
 !@
 
-### Button Presses
-Another unique feature of remote control is the ability to send simulated button presses to the associated modules, imitating a button press on the hardware itself. Simply specify the module, the button, and the type of press you would like.
+#### Button Presses
+Another unique feature of remote control is the ability to send simulated button presses to the associated modules, imitating a button press on the hardware itself. Simply specify the module, the button, and the type of press you would like to simulate.
 
 @![iOS]
 ##### Objective-C
+###### RPC < v6.0
 ```objc
 SDLButtonPress *buttonPress = [[SDLButtonPress alloc] initWithButtonName:SDLButtonNameEject moduleType:SDLModuleTypeRadio];
 buttonPress.buttonPressMode = SDLButtonPressModeShort;
 
-[self.sdlManager sendRequest:buttonPress];
+[self.sdlManager sendRequest:buttonPress withResponseHandler:^(__kindof SDLRPCRequest * _Nullable request, __kindof SDLRPCResponse * _Nullable response, NSError * _Nullable error) {
+    if(!response.success) { return; }
+}];
 ```
 
+###### RPC v6.0+
+```objc
+SDLButtonPress *buttonPress = [[SDLButtonPress alloc] initWithButtonName:SDLButtonNameEject moduleType:SDLModuleTypeRadio moduleId:@"<#ModuleID#>"];
+buttonPress.buttonPressMode = SDLButtonPressModeShort;
+
+[self.sdlManager sendRequest:buttonPress withResponseHandler:^(__kindof SDLRPCRequest * _Nullable request, __kindof SDLRPCResponse * _Nullable response, NSError * _Nullable error) {
+    if(!response.success) { return; }
+}];
+``` 
+
 ##### Swift
+###### RPC < v6.0
 ```swift
 let buttonPress = SDLButtonPress(buttonName: .eject, moduleType: .radio)
 buttonPress.buttonPressMode = .short
 
-sdlManager.send(buttonPress)
+sdlManager.send(request: buttonPress) { (request, response, error) in
+    guard response?.success.boolValue == true else { return }
+}
+```
+
+###### RPC v6.0+
+```swift
+let buttonPress = SDLButtonPress(buttonName: .eject, moduleType: .radio, moduleId: "<#ModuleID#>")
+buttonPress.buttonPressMode = .short
+
+sdlManager.send(request: buttonPress) { (request, response, error) in
+    guard response?.success.boolValue == true else { return }
```

Comment:
```suggestion
    guard response.resultCode == .success else { return }
```

Reply:
We should be checking `success`, not the `resultCode` (e.g. WARNING shouldn't fail it).